libs/cdk/src/lib/render-strategies/utils.ts

```diff
@@ -0,0 +1,83 @@
+import { map, materialize, tap } from 'rxjs/operators';
+import { Notification, Observable, of, OperatorFunction } from 'rxjs';
+import {
+  RenderWork,
+  StrategyCredentials,
+  StrategyCredentialsMap,
+  RxNotification,
+  RxNotificationKind,
+  RxErrorNotification,
+  RxCompleteNotification
+} from './model';
+import { CoalescingOptions } from '../model';
+
+
+const rxJsToRxA = {
```

Comment:
we should turn on `strict mode`

Reply:
```suggestion
const rxJsToRxA: Record<'N' | 'E' | 'C', RxNotificationKind> = {
```

https://github.com/ReactiveX/rxjs/blob/master/src/internal/Notification.ts

`NotificationKind` is `@deprecated`